File: Linked Lists/mergeTwoLists.js
class ListNode {
    constructor(val) {
        this.val = val
        this.next = null
    }
}

const mergeTwoLists = (l1, l2) => {
    let current = new ListNode(0)
    let newList = current
    while (l1 && l2) {
        if (l1.val < l2.val) {
            current.next = l1
            l1 = l1.next
        } else {
            current.next = l2
            l2 = l2.next
        }
        current = current.next
    }
    current.next = l1 || l2
    return newList.next
}

File: Linked Lists/mergeTwoLists.py
class ListNode:
    def __init__(self, val):
        self.val = val
        self.next = None

class Solution:
    def mergeTwoLists(self, l1: ListNode, l2: ListNode) -> ListNode:
        newList = current = ListNode(0)
        while l1 and l2:
            if l1.val < l2.val:
                current.next = l1
                l1 = l1.next
            else:
                current.next = l2
                l2 = l2.next
            current = current.next
        current.next = l1 or l2
        return newList.next